ca/ca.go

```diff
@@ -398,7 +398,7 @@ func (ca *CertificateAuthorityImpl) IssueCertificate(ctx context.Context, csr x5
 	notAfter := ca.clk.Now().Add(ca.validityPeriod)
 
 	if issuer.cert.NotAfter.Before(notAfter) {
-		err = core.InternalServerError("Cannot issue a certificate that expires after the issuer certificate.")
+		err = berrors.MalformedError("cannot issue a certificate that expires after the issuer certificate")
```

Comment:
This should stay as an InternalServerError, since nothing should get this far without a mistake on our part.